workers/Managed/build.json

```diff
@@ -42,19 +42,20 @@
           ]
         },
         {
-          "name": "CMake build",
+          "name": "Project files from CMake",
           "working_path": "cmake_build",
           "command": "bash",
           "arguments": [
-            "../build.sh"
+            "../cmake_configure.sh"
           ]
         },
         {
-          "name": "Zip worker",
+          "name": "Worker zip",
           "working_path": "cmake_build",
           "command": "cmake",
           "arguments": [
             "--build", ".",
+            "--config", "Release",
```

Comment:
README.md still refers to us building /MDd by default - do we need to alter the readme to reflect this?

Reply:
Ah good spot. Will fix.